docs/static/config-management.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[config-management]]
+== Configuration Management
+
+Logstash provides configuration management features to make it easier for you to
+manage updates to your configuration over time.
+
+[role="xpack"]
+[[logstash-centralized-pipeline-management]]
+=== Centralized Pipeline Management
+
+//REVIEWERS: I'm not sure if centralized pipeline management is the right term to use here, but if we want to have a generic container for lots of types of configuration management, we need to distinguish pipeline management.
+
+NOTE: Centralized pipeline management is an {xpack} feature that requires a
+paid {xpack} license. 
+
+//REVIEWERS: I'm not sure we want to mention actual types of licenses, like Gold, if they might change in the future. Is there a resource that I can point to that outlines the feature support? Is there other language that I should use here to call out the fact that a paid license is required?
+
+The pipeline management feature in {xpack} centralizes the creation and
+management of Logstash configuration pipelines. From within the pipeline
+management UI, you can add, edit, and delete pipeline configurations.
+
+The pipeline configurations are stored in the .`logstash` index in Elasticsearch.
+Any changes that you make to a pipeline definition in the UI are picked up and
+loaded automatically by all Logstash instances registered to use the pipeline.
+The changes are applied immediately; you do not have to restart Logstash to pick
+up the changes, as long as Logtash is already registered to use the pipeline.
```

Comment:
Also: 

Once you use centralized config management feature, you will not be able to specify local pipeline configuration. This means `pipelines.yml` and settings like `path.config` `config.string` will be inactive when this feature is used.